app/presenters/start_node_presenter.rb

```diff
@@ -19,6 +19,11 @@ def body
     @renderer.content_for(:body)
   end
 
+  def post_body_header
+    custom_header_text = @renderer.content_for(:post_body_header)
+    custom_header_text.presence || "Before you start"
```

Comment:
This is nice! I think that the 23 other smart-answers remain unaffected.

Reply:
Yeah, we tried not to mess with any of the other flows